contrib/Shopify/sarama/sarama.go

```diff
@@ -35,6 +36,7 @@ func WrapPartitionConsumer(pc sarama.PartitionConsumer, opts ...Option) sarama.P
 	for _, opt := range opts {
 		opt(cfg)
 	}
+	log.Debug("contrib/Shopify/sarama: Wrapping Partition Consumer: %#v", cfg)
```

Comment:
I'm going back and forth about printing the config with `%#v`. It's easy and a decent aproximation in most cases, so I think it might be ok.

Let me know any thoughts.

Reply:
Absolutely. Why not?